packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -141,22 +141,22 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
         }
 
         const onSyncGlobals = ({ config, env }) => {
-          syncConfigToCurrentDomain(config)
-          syncEnvToCurrentDomain(env)
+          syncConfigToCurrentOrigin(config)
+          syncEnvToCurrentOrigin(env)
         }
 
         communicator.once('sync:globals', onSyncGlobals)
 
-        communicator.once('ran:domain:fn', (details) => {
+        communicator.once('ran:origin:fn', (details) => {
```

Comment:
Newbie, what then does `ran:orgin:fn` occur within the cy.orgin event order?

Reply:
`ran:origin:fn` occurs after `run:origin:fn` lol
It lets us know that we've run the code in the cy.origin block, primed the command queue for the specbridge and we're ready for the server to let the cross origin site load and for us to then run the command queue

Reply:
yes